test/unittests/t_util.cc

```diff
@@ -1763,3 +1764,57 @@ TEST_F(T_Util, DiffTree) {
   EXPECT_TRUE(DiffTree("./.", "."));
   EXPECT_FALSE(DiffTree(".", "/"));
 }
+
+TEST(BoundaryValueTest, 2Bin_Empty) {
```

Comment:
Let's rename to `TEST(Log2Histogram, 2BinEmpty)`

Reply:
Done!